include/net/aws_fota.rst

```diff
@@ -64,6 +63,33 @@ The implementation uses a job document similar to the following (where *bucket_n
 
 The current implementation uses information from the ``host`` and ``path`` fields only.
 
+For upgrading MCUBoot you'll have to do a similar process as the above except you'll have to turn on `CONFIG_SECURE_BOOT` and `CONFIG_MCUBOOT_BUILD_S1_VARIANT` for your application this will result in two files being generated when building your application which ends in *s0/s1_image_update.bin* in the build folder of your application. The URL of these files are added to the path starting with the *s0* file followed by the *s1* file where the path is separated by a space.
+
+.. note::
+  It is important that you place the path of *s0* before *s1* in the job document.
+
+
+.. parsed-literal::
+   :class: highlight
+
+   {
+     "operation": "bootloader_fw_update",
+     "fwversion": "v1.0.2",
+     "size": 241124,
+     "location": {
+       "protocol": "http:",
+       "host": "*bucket_name*.amazonaws.com",
+       "path": "*s0*.bin *s1*.bin"
+      }
+   }
+
+
+.. warning::
+  Not available in NCS 1.2
```

Comment:
We're not talking about nRF Cloud here at all, but about plain AWS, so this warning is confusing ... (to me at least ;-) )

Reply:
Should this go into the sample? I'm not sure where to document that this feature is not available in nRF Cloud at the moment but will be added at a later stage?

Reply:
We're not mentioning nRF Cloud in the sample either ... (or rather, just as an option for hosting, but not for updating)